contrib/pam_zfs_key/pam_zfs_key.c

```diff
@@ -642,10 +661,28 @@ pam_sm_authenticate(pam_handle_t *pamh, int flags,
 {
 	(void) flags, (void) argc, (void) argv;
```

Comment:
now `argc` and `argv` are not unused any longer, so you can remove the `(void)`-cast here, right?